usr/src/uts/intel/io/i915/i915_dma.c

```diff
@@ -1510,7 +1510,7 @@ int i915_driver_load(struct drm_device *dev, unsigned long flags)
 out_rmmap:
 	drm_ioremapfree(dev_priv->regs);
 put_bridge:
-free_priv:
+/* free_priv: */
```

Comment:
Much of this code is tracking an upstream, so I though it
might help me do that if I leave the labels commented.